manifest.json

```diff
@@ -97,8 +97,20 @@
                 {
                     "name": "modelGeneratorGitRef",
                     "type": "string",
-                    "default": "v0.6.0",
+                    "default": "v0.7.0",
                     "description": "Git ref of the model generator repo. Can be a tag, branch or SHA"
+                },
+                {
+                    "name": "vssSrc",
+                    "type": "string",
+                    "default": "https://github.com/COVESA/vehicle_signal_specification/releases/download/v3.0/vss_rel_3.0.json",
```

Comment:
I do not think. We didn't test thoroughly but it should work as expected since databroker is supporting the general syntax and would work. Starting 0.3.0 with the default VSS 4.0 works.